dev-aws/account-platform/account.tf

```diff
@@ -0,0 +1,14 @@
+# tflint-ignore: terraform_naming_convention
```

Comment:
Since you're just starting with this ns, I'd recommend fixing the resource names from the start. It is harder to fix after the first apply.

Reply:
Haha, was copy and. paste, will check the correct format

Reply:
it just needs to be `_` instead of `-`